web/src/components/dashboard/DashboardAlerts.tsx

```diff
@@ -23,6 +24,16 @@ export const DashboardAlerts = (): ReactElement => {
     dataTestId: "snackbar-alert-calendar",
   });
 
+  const CheckboxStatusUpdated = useQueryControlledAlert({
+    queryKey: QUERIES.fromFormBusinessEntity,
+    pagePath: ROUTES.dashboard,
+    headerText: Config.taskDefaults.taskProgressSnackbarHeading,
+    bodyText: getTaskStatusUpdatedMessage("COMPLETED"),
+    variant: "success",
+    dataTestId: "checkbox-status-updated-snackbar-alert",
+    delayInMilliseconds: Config.dashboardDefaults.timeDelayBetweenMultipleSnackbars,
```

Comment:
[dust] I might set this at the top and reference it below.

```typescript
const delayBetweenMultipleSnackbars = Config.dashboardDefaults.timeDelayBetweenMultipleSnackbars;
 

const CheckboxStatusUpdated = useQueryControlledAlert({
  ...
  delayInMilliseconds: delayBetweenMultipleSnackbars;
});


const DeadlinesAndOpportunitiesAlert = useQueryControlledAlert({
  ...
  delayInMilliseconds: delayBetweenMultipleSnackbars;
});
```

Reply:
That makes sense